config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'newsletters/create'
```

Comment:
looks like unnecessary line

Reply:
good point, I think I just missed this when refactoring